docs/getting-started-publishing.md

```diff
@@ -73,36 +73,42 @@ However, you can automate the publishing process with continuous integration (CI
 
 Continuous integration (CI) services are typically used to perform routine tasks whenever new commits are checked in to source control. These tasks can be any combination of running unit tests and integration tests, automating builds, publishing packages to NPM, and yes, deploying changes to your website. All you need to do to automate deployment of your website is to invoke the `publish-gh-pages` script whenever your docs get updated. In the following section we'll be covering how to do just that using [Circle CI](https://circleci.com/), a popular continuous integration service provider.
 
-### Using Circle CI
+### Using Circle CI 2.0
 
 If you're already using Circle CI for your project, all you need to do to enable automatic deployments is to configure Circle to run the `publish-gh-pages` script as part of the deployment step.
 
+If not, here's how to setup Circle CI for your project:
```

Comment:
I think we want people to run through these steps, even if they are already using CircleCI for their main open source project. I made a suggested change to the above text to make that clearer.